practices/graded-browser-support.md

```diff
@@ -9,27 +9,34 @@
 We follow the guidelines for graded browser support outlined by Nate Koechley at Yahoo:
 [Yahoo Grade Browser Support guide](https://github.com/yui/yui3/wiki/Graded-Browser-Support)
 
-| Tables        | Grade-A support                 | Grade-C support |
-| ------------- |:-------------------------------:| ---------------:|
-| Chrome        | latest stable                   |                 |
-| Edge          | latest stable                   |                 |
-| Firefox       | latest stable                   |              <2 |
-| IE            | 11                              |             <11 |
-| Opera         | latest stable                   |             <10 |
-| Safari        | latest stable, latest stable -1 |  <4.1 (desktop) |
-| Webkit        | Android 4.&#8224;               |              <3 |
+Our browser support in summary:
+
+- Grade-A are browser versions that are explicitly *supported*. A product bug in one of these browsers is a high priority.
+- Grade-C are browser versions that are explicitly *_not_ supported*. We serve only the minimum styling and functionality to these browsers. 
+- Grade-X are the grey area between Grade-A and Grade-C. We serve them the same code as Grade-A, but we do not test our products against these browser versions. Over time, analysis of errors and usage patterns may cause some browsers in Grade X to be deliberately demoted to Grade C. 
+
+| Tables                      | Grade-A support                 | Grade-C support    |
+| --------------------------- |:-------------------------------:| ------------------:|
+| Chrome                      | latest stable, latest stable -1 | < 29               |
```

Comment:
Doesn't "Chrome 29+" mean "29 and above"? If so, I've used "< 29" to mean "less than 29".